test/typescript/I18nextProvider.test.tsx

```diff
@@ -23,7 +23,7 @@ i18next.use(initReactI18next).init({
 function test() {
   return (
     <React.Suspense fallback={<p>Loading</p>}>
-      <I18nextProvider i18n={i18next}>
+      <I18nextProvider i18n={i18next} defaultNS={'translations'}>
```

Comment:
Please don't change the current test because it is a valid case on it's own.  Please add a new `function testDefaultNS()`